app/src/main/java/ru/chaichuk/hwapp/viewModels/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,17 @@
+package ru.chaichuk.hwapp.viewModels
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import ru.chaichuk.hwapp.data.Movie
+
+class MovieDetailsViewModel : ViewModel() {
+
+    private val _mutableMovie = MutableLiveData<Movie>()
+
+    val movie: LiveData<Movie> get() = _mutableMovie
+
+    fun loadMovie (newMovie : Movie) {
```

Comment:
very strange method 
why we need like so the method 
which problem it solve ?

Reply:
where he used

Reply:
it is called in onViewCreated for MoviesDetailsFragment, and I pass to it movie data from arguments, which I got from main activity